requirements.txt

```
@@ -1,3 +1,6 @@
 django>=4.0, <5.0
 gunicorn
-django-heroku
\ No newline at end of file
+pytest
+dj-database-url>=0.5.0
+whitenoise
+psycopg2
```

I don't think `pytest` was installed before - unless we need it here for some reason, I'd lean towards omitting it? 

Also, whilst django-heroku didn't do this, I think we might want to start adding version ranges to all of the packages, so they don't suddenly start breaking if a new major version is released. We can always also enable Dependabot for this repo (I'll file a new GUS work item for that shortly).

```suggestion
dj-database-url>=0.5,<0.6
whitenoise>=6.0,<7.0
psycopg2>=2.0,<3.0
```

---

made change. tested by create venv and installing to make sure everything installs properly. will make change to the guide draft as well